var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -59,9 +59,6 @@ class Gcc(AutotoolsPackage):
     variant('binutils',
             default=sys.platform != 'darwin',
             description="Build via binutils")
-    variant('gold',
-            default=sys.platform != 'darwin',
-            description="Build the gold linker plugin for ld-based LTO")
```

Comment:
I agree.  Especially now that we  can set this stuff up in `packages.yaml`  (putting `^` on the command line doesn't always work intuitively).